i18n/es.po

```diff
@@ -1,64 +1,145 @@
-# Traducción en Español
-# Copyright (C) 2013 ORGANIZATION
+# Translations template for PROJECT.
+# Copyright (C) 2019 ORGANIZATION
 # This file is distributed under the same license as the PROJECT project.
-# Raul Gallegos <raul.ogh@gmail.com>, 2013.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
+#
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2013-05-02 18:52-0500\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: Raul Gallegos <raul.ogh@gmail.com>\n"
-"Language-Team: Spanish <LL@li.org>\n"
+"POT-Creation-Date: 2019-07-15 18:44-0500\n"
+"PO-Revision-Date: 2019-07-16 10:10-0500\n"
+"Language-Team: \n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 0.9.6\n"
+"Generated-By: Babel 2.7.0\n"
+"X-Generator: Poedit 2.2.3\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: es\n"
 
-#: templates/base.html:31
+#: templates/base.html:30
 msgid "Main Page"
 msgstr "Página Principal"
 
+#: templates/base.html:31 templates/features.html:3 templates/features.html:7
+msgid "Features"
+msgstr "Características"
+
 #: templates/base.html:32 templates/download.html:4
 msgid "Download"
-msgstr "Descarga"
-
-#: templates/base.html:33 templates/docs.html:4 templates/docs.html:8
-#: templates/index.html:199
+msgstr "Descargar"
```

Comment:
I think that "Download" is imperative form. Probably "Donate" and "Calculate" as well.

Reply:
> In Spanish, using the infinitive is correct,

I take your word for that :)